scrapegraphai/nodes/fetch_node.py

```diff
@@ -32,6 +34,7 @@ class FetchNode(BaseNode):
     def __init__(self, input: str, output: List[str], node_config: Optional[dict]=None, node_name: str = "Fetch"):
         super().__init__(node_name, "node", input, output, 1)
 
+        self.useSoup = True if node_config is None else node_config.get("useSoup", True)
```

Comment:
using the beautiful soup by default since it is a lot more comprehensible compared to AsyncChromiumLoader. In future, we can augment this node further to fetch more detailed information when both beautiful soup and AsyncChromiumLoader approach does not work